src/main/webapp/app/core/posthog/analytics.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { ProfileInfo } from 'app/shared/layouts/profiles/profile-info.model';
+import { posthog } from 'posthog-js';
```

Comment:
I am not 100% sure, but couldn't we use a conditional import so that we reduce the js shipped to the client when posthog is not used?

```
if (condition) {
    import('something')
    .then((something) => {
       console.log(something.something);
    });
}
```

Reply:
Fair point - I'll check!

Reply:
Let's check this in a follow-up